### PR TITLE
[zh-cn] Localize generated/kubectl_attach

### PR DESCRIPTION
**Pull request by my-git9**
content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md

### CODE REVIEW

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+auto_generated: true
+no_list: true
+-->
+
+## {{% heading "synopsis" %}}
+
+<!--
+Attach to a process that is already running inside an existing container.
+-->
+附加到现有容器内已运行的进程。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md:20]:
```suggestion
挂接到现有容器内已运行的进程。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+
+# Get output from ruby-container from pod mypod
+
+# Switch to raw terminal mode; sends stdin to 'bash' in ruby-container from pod mypod
+# and sends stdout/stderr from 'bash' back to the client
+
+# Get output from the first pod of a replica set named nginx
+-->
+```
+  # 从运行的 Pod mypod 获取输出；使用 'kubectl.kubernetes.io/default-container' 注解来选择要附加的容器，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md:40]:
```suggestion
  # 从运行的 Pod mypod 获取输出；使用 'kubectl.kubernetes.io/default-container' 注解来选择要挂接的容器，
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md [hunk 3]
```diff
@@ -0,0 +1,607 @@
...
+  kubectl attach mypod
+  
+  # 从 Pod mypod 获取 ruby-container 的输出
+  kubectl attach mypod -c ruby-container
+  
+  # 切换到原始终端模式；从 Pod mypod 将 stdin 发送到 ruby-container 中的 'bash'，
+  # 并将 stdout/stderr 从 'bash' 发送回客户端
+  kubectl attach mypod -c ruby-container -i -t
+  
+  # 获取名为 nginx 的副本集的第一个 Pod 的输出
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md:51]:
```suggestion
  # 获取名为 nginx 的 ReplicaSet 中第一个 Pod 的输出
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md [hunk 4]
```diff
@@ -0,0 +1,607 @@
...
+<td colspan="2">-c, --container string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;">
+<p>
+<!--
+Container name. If omitted, use the kubectl.kubernetes.io/default-container annotation for selecting the container to be attached or the first container in the pod will be chosen
+-->
+容器名称。
+如果省略，则使用 kubectl.kubernetes.io/default-container 注解来选择要附加的容器，
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md:74]:
```suggestion
如果省略，则使用 kubectl.kubernetes.io/default-container 注解来选择要挂接的容器，
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md [hunk 5]
```diff
@@ -0,0 +1,607 @@
...
+
+<tr>
+<td colspan="2">--cluster string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+<!--
+The name of the kubeconfig cluster to use
+-->
+kubeconfig 中要使用的集群的名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md:278]:
```suggestion
要使用的 kubeconfig 中集群的名称。
```

File: content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md [hunk 6]
```diff
@@ -0,0 +1,607 @@
...
+
+<tr>
+<td colspan="2">--context string</td>
+</tr>
+<tr>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>
+<!--
+The name of the kubeconfig context to use
+-->
+kubeconfig 要使用的上下文的名称。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubectl/generated/kubectl_attach/_index.md:290]:
```suggestion
要使用的 kubeconfig 上下文的名称。
```

**Review by ydFu — APPROVED** [review]
/approve

**Pull request merged**